main.py

```diff
@@ -1,13 +1,8 @@
-from logic import *
-
-#Start your program here
-
-a = 1
-
-def b():
-    global a
-    a += 2
-    print( " " + str(a))
-
-b()
-print(a)
\ No newline at end of file
+while True:
+    number_1 = int(input("what is the number 1 "))
+    number_2 = int(input("what is the number 2 "))
+    print(number_1 + number_2)
```

Comment:
The previous 3 lines do the following:

Read input
Read input
Print some result

It's often preferred to categorize these into sections by adding a new line between them. Also, adding some comments would be nice. E.g.

```
#Read the two numbers from the user
number_1 = int(input("what is the number 1 "))
number_2 = int(input("what is the number 2 "))

#Print the result
print(number_1 + number_2)
```